src/tools/rbd_mirror/group_replayer/Replayer.cc

```diff
@@ -1181,7 +1241,7 @@ void Replayer<I>::post_mirror_snapshot_complete(
       local_snap != m_local_group_snaps.end(); ++local_snap) {
     if (local_snap->id == group_snap_id) {
       break;
-    } else if (local_snap->state == cls::rbd::GROUP_SNAPSHOT_STATE_INCOMPLETE) {
+    } else if (local_snap->state == cls::rbd::GROUP_SNAPSHOT_STATE_CREATING) {
       locker.unlock();
       // there is a user group snap waiting sync, so lets wait for it to moved to complete
```

Comment:
This comment and one above it still mention moving to complete in the context of a user snapshot.